### DISCUSSION TITLE
how to convert data to `sktime` format for shapelet transform

### DISCUSSION THREAD

**Discussion by fkiraly** [alan-turing-institute/sktime#2011] [Q&A]
from gitter, @aron-alarik:

"Hey !! I have a univariate series that I am trying convert to sktime format, from Pandas, as mentioned in the documentation, want to use this with shapelet transform...thanks !! 

![image](https://user-images.githubusercontent.com/7985502/152705111-013d18fd-32b6-4ed5-8994-c75e44aeb2de.png)

this is what my df looks like :) 

![image](https://user-images.githubusercontent.com/7985502/152705120-30d143e7-589f-49fc-b37f-ed99d6261f0b.png)

**Comment by fkiraly** [comment]
@aron-alarik, I think this is very close - it looks like an ordinary DataFrame with an index row? Then, you'd just need to do `X_nested.set_index("time")`, this will give you a single univariate series in a `pd.DataFrame`

**Reply by aron-alarik** [reply]
Thanks ! this is very helpful, let me try this out and get back to you.
